runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/streaming/harness/StreamingWorkerStatusReporterTest.java

```diff
@@ -59,18 +59,19 @@ public void setUp() {
   @Test
   public void testOverrideMaximumThreadCount() throws Exception {
     StreamingWorkerStatusReporter reporter =
-        StreamingWorkerStatusReporter.forTesting(
-            true,
-            mockWorkUnitClient,
-            () -> DEFAULT_WINDMILL_QUOTA_THROTTLE_TIME,
-            () -> Collections.emptyList(),
-            mockFailureTracker,
-            StreamingCounters.create(),
-            mockMemoryMonitor,
-            mockExecutor,
-            (threadName) -> Executors.newSingleThreadScheduledExecutor(),
-            DEFAULT_HARNESS_REPORTING_PERIOD,
-            DEFAULT_PER_WORKER_METRICS_PERIOD);
+        StreamingWorkerStatusReporter.builder()
```

Comment:
can you make a testReporterBuilder() in this test that sets all these defaults?

Reply:
done